### DISCUSSION TITLE
A NoSQL database?

### DISCUSSION THREAD

**Discussion by fuzzypixelz** [kdl-org/kdl#282] [Applications]
[XML](https://en.wikipedia.org/wiki/XML_database) databases are well known but not widely used, they fit a niche domain. But reading the spec for the KDL Query Language made me think "it would be fun to run with that and make an actual database".

Have you ever considered such an idea? Is it too crazy? Thoughts?

**Comment by zkat** [comment]
That sounds like a fun project!
